packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/NextButton/NextButton.js

```diff
@@ -9,21 +8,16 @@ import en from './locales/en'
 
 counterpart.registerTranslations('en', en)
 
-const Label = (
-  <Text size='medium'>
-    {counterpart('NextButton.next')}
-  </Text>
-)
-
 function NextButton (props) {
   const { autoFocus, disabled, onClick } = props
+  const label = counterpart('NextButton.next')
   return (
-    <Button
-      a11yTitle={counterpart('NextButton.next')}
+    <GoldButton
+      a11yTitle={label}
```

Comment:
This shouldn't duplicate the button label. It looks like you can remove it, since the button has a visible label.